website/docs/docs/dbt-versions/release-notes/09-April-2023/scheduler-optimized.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "dbt Cloud more intelligently processes scheduled jobs"
```

Comment:
non-blocker but i think moving the verb before the adverb sounds nicer and avoids splitting the infinitive 

```suggestion
title: "dbt Cloud processes scheduled jobs more intelligently"
```